packages/cf-component-toggle/src/ToggleTheme.js

```diff
@@ -0,0 +1,46 @@
+export default baseTheme => ({
+  display: 'block',
+  position: 'relative',
+  margin: 0,
+  height: '2.26667rem',
+  width: '5.334rem',
+  borderRadius: '2px',
+  borderWidth: '1px',
+  borderStyle: 'solid',
+  borderColor: '#aaa',
```

Comment:
I dunno, I took it directly from the CSS and [am unsure on how to find a equivilent one](https://github.com/cloudflare/cf-ui/issues/239).